### PR TITLE
Update qit-cli to 0.4.0

### PR DESCRIPTION
**Pull request by kloon**
## Description

Our qit tests as failing since the introduction of 0.4.0 and qit moving to a new domain

### Steps to reproduce & screenshots/GIFs
Verify QIT tests pass on this PR.

### Checklist
This is only a development change so does not need a release or version bump.

- [ ] unit tests

- [ ] `changelog.txt` entry added
- [ ] `readme.txt` entry added

### CODE REVIEW

**Comment by kloon** [comment]
I'll take a look at fixing the PHP tests in another PR. Those are related to WC core that switched their build process in 8.5 https://github.com/woocommerce/woocommerce/pull/41988

**Review by iyut — APPROVED** [review]
The changes does fix the QIT error. LGTM!

File: composer.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
     "require-dev": {
       "woocommerce/woocommerce-sniffs": "^0.1.3",
       "phpunit/phpunit": "^9.6.8",
-      "woocommerce/qit-cli": "^0.3.4",
       "squizlabs/php_codesniffer": "^3.7.2",
       "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
-      "wp-coding-standards/wpcs": "^2.3"
+      "wp-coding-standards/wpcs": "^2.3",
+        "woocommerce/qit-cli": "^0.4.0"
```
Comment (iyut) [inline composer.json:8]: Just out of curiosity, may I know the reason why the `woocommerce/qit-cli` is moved to the bottom?

Comment (kloon) [reply composer.json:8]: I am not sure, I just used the composer command to update it from the terminal.

**Pull request merged**